src/Core/Components/Stack/FluentStack.razor.cs

```diff
@@ -74,6 +77,24 @@ public partial class FluentStack : FluentComponentBase
     [Parameter]
     public RenderFragment? ChildContent { get; set; }
 
+    protected override void OnParametersSet()
+    {
+        if (ChildContent != _cachedContent)
+        {
+            _cachedContent = ChildContent;
+        }
```

Comment:
I still continue to have `_shouldRender = true;`: when the ChildContent changes later, the HTML result must be adapted.